trajectorytools/trajectories.py

```diff
@@ -322,7 +322,9 @@ def __str__(self):
 
     @classmethod
     def from_idtrackerai(cls, trajectories_path, **kwargs):
-        return cls.from_idtracker(trajectories_path, **kwargs)
+        tr = cls.from_idtracker(trajectories_path, **kwargs)
+        tr.params["construct_method"] = "from_idtrackerai"
+        return tr
```

Comment:
Maybe remove the annotation here? from_idtrackerai is just a synonym of from_idtracker.

Alternatively, if we like "from_idtrackerai" more, we can have "from_idtrackerai" as the main and the other as an alias. We can add a deprecation warning if we feel like removing "from_idtracker" in the future